src/napari_workflow_inspector/_dock_widget.py

```diff
@@ -118,48 +154,13 @@ def html(text):
 
             lbl_from_roots.setText(html(build_output(workflow.roots(), workflow.followers_of)))
 
-            if len(self._edges) > 0:
-
-                import igraph
-                idfg = igraph.Graph(self._edges)
-
-                def redraw():
-                    igraph.plot(idfg,
-                                target=igraphwidget.axes,
-                                vertex_label=["   " + name for name in self._names],
-                                vertex_color=self._statii,
-                                vertex_label_color = self._statii,
-                                vertex_size = 10,
-                                vertex_label_dist = 10,
-                                vertex_label_size = 10,
-                                edge_color = "#dddddd",
-                                layout=self.graph_layout)
-
-
-                if not (np.array_equal(self.idfg_edges, np.asarray(self._edges)) and self.idfg_statii == str(self._statii) and self.idfg_names == str(self._names)):
-
-                    if self.graph_layout is None:
-                        self.graph_layout = idfg.layout_auto()
-
-                    # workaround as vertex_label_color above doesn't work :-(
-                    matplotlib.rcParams['text.color'] = '#dddddd'
```

Comment:
I _think_ also this line should be kept somewhere to make the text readable on dark background.